deploy/crd/exposedsecretreports.crd.yaml

```diff
@@ -0,0 +1,229 @@
+                      type: integer
+                      minimum: 0
+                secrets:
+                  description: |
+                    Exposed secrets is a list of passwords, api keys, tokens and others items found in the Artifact.
+                  type: array
+                  items:
+                    type: object
+                    required:
+                      - target
+                      - ruleID
+                      - title 
+                      - category 
+                      - severity
+                      - match
+                    properties:
+                      target:
+                        description: |
+                          Target is where the exposed secret was found.
+                        type: string
+                      ruleID:
+                        description: |
+                          RuleID is rule the identifier.
+                        type: string
+                      title:
+                        type: string
+                      category:
+                        type: string
+                      severity:
+                        type: string
+                        enum:
+                          - CRITICAL
+                          - HIGH
+                          - MEDIUM
+                          - LOW
+                          - UNKNOWN
+                      match:
+                        description: |
+                          Match where the exposed rule matched.
+                        type: string
+      additionalPrinterColumns:
+        - jsonPath: .report.artifact.repository
+          type: string
+          name: Repository
+          description: The name of image repository
+        - jsonPath: .report.artifact.tag
+          type: string
+          name: Tag
+          description: The name of image tag
+        - jsonPath: .report.scanner.name
+          type: string
+          name: Scanner
+          description: The name of the exposed secret scanner
+        - jsonPath: .metadata.creationTimestamp
+          type: date
+          name: Age
+          description: The age of the report
+        - jsonPath: .report.summary.criticalCount
+          type: integer
+          name: Critical
+          description: The number of critical exposed secrets
+          priority: 1
+        - jsonPath: .report.summary.highCount
+          type: integer
+          name: High
+          description: The number of high exposed secrets
+          priority: 1
+        - jsonPath: .report.summary.mediumCount
+          type: integer
+          name: Medium
+          description: The number of medium exposed secrets
+          priority: 1
+        - jsonPath: .report.summary.lowCount
+          type: integer
+          name: Low
+          description: The number of low exposed secrets
+          priority: 1
+        - jsonPath: .report.summary.unknownCount
```

Comment:
Not sure if severity `unknown` is relevant to secret world. its more related vulnerability scaninng  when it comes from an external DB and its, in secret we define the severity so unknow not likely to happen